extensions/shared/components/stripe-nudge/style.scss

```diff
@@ -11,3 +11,8 @@
 	margin-right: 16px;
 	padding: 6px;
 }
+
+.jetpack-block-nudge.block-editor-warning .block-editor-warning__contents {
```

Comment:
This will target all usages of [the `BlockNudge` component](https://github.com/Automattic/jetpack/blob/42c9668c752128fd82d948fd56decf61010e3854/extensions/shared/components/block-nudge/index.jsx#L27). Is this intended? Or should we specifically target the Stripe nudges only?

Reply:
Good catch! We don't appear to have a class to target Stripe warnings specifically, but we could use the individual block wrapper names, or add a unique class. I'll do a quick check to see where else we show block warnings in Jetpack to see if the styles work to be safe.